src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -328,6 +336,53 @@ private async Task<string> GetPsVersionInfo()
             return builder.ToString();
         }
 
+        private async Task<string> GetLocalGroupMembership(string userName)
+        {
+            var builder = new StringBuilder();
+
+            string powerShellExe = HostContext.GetService<IPowerShellExeUtil>().GetPath();
```

Comment:
Do we probably need some fallback to previous behavior with try/catch block - so if there is an error, it won't crush agent, but just write some message in logs about the exception?